.gitignore

```diff
@@ -44,4 +38,3 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
-.firebase
```

Comment:
We want to leave the `.firebase` as this folder will be created during deploy, which is currently done manually.